.vsts-ci.yml

```diff
@@ -3,7 +3,11 @@ queue: 'Hosted VS2017'
 variables:
   buildConfiguration: 'Release'
 
-steps:
+steps:steps:
```

Comment:
this is the reason the ci build fails:
![fail](https://user-images.githubusercontent.com/12570757/56970003-885ad080-6b66-11e9-9d37-d7f0e45ce5fe.png)

Reply:
Good catch. Looks like a dumb copy-paste mistake on my part.

Reply:
Also had to update the version, but it seems that Azure DevOps doesn't have the latest preview yet?

Reply:
You need to add the Install .Net Core SDK task. Check the SignarlR project here on both CI and Release yaml files.

Reply:
Thanks for the reference. I see my mistakes now. Corrected.